partner_multi_relation/models/res_partner_relation_type.py

```diff
@@ -84,16 +84,6 @@ def get_partner_types(self):
             ('p', _('Person')),
         ]
 
-    @api.onchange('is_symmetric')
-    def onchange_is_symmetric(self):
-        """Set right side to left side if symmetric."""
-        if self.is_symmetric:
-            self.update({
-                'name_inverse': self.name,
-                'contact_type_right': self.contact_type_left,
-                'partner_category_right': self.partner_category_left,
-            })
-
```

Comment:
What where the reasons to  remove the onchange and replace it with a write/create overwrite ?

Reply:
@gfcapalbo Using write/create means that values will be computed the correct way, even if relations inserted/modified through back-end. For instance on import.